src/ElevatorButton.java

```diff
@@ -0,0 +1,16 @@
+public class ElevatorButton {
+  FloorLamp lamp;
+
+  public ElevatorButton() {
+    lamp = new FloorLamp(false);
```

Comment:
If you guys are also using FloorLamp then the class should be renamed to "ButtonLamp"